.github/scripts/generate_binary_build_matrix.py

```diff
@@ -153,26 +151,17 @@ def initialize_globals(channel: str, build_python_only: bool) -> None:
         XPU: "pytorch/manylinux2_28-builder:xpu",
         # TODO: Migrate CUDA_AARCH64 image to manylinux2_28_aarch64-builder:cuda12.4
```

Comment:
change this comment to manylinux2_28_aarch64-builder:cuda12.6 if that's what you're using